samples/samples-js/AddProductWithIdentityColumn/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const itemToInsert = {
+        "name": req.params?.name ?? "productWithIdentity",
```

Comment:
Don't have defaults, we can add that functionality in later if we think it's necessary